### PR TITLE
Addressing docker scout findings

### PR DESCRIPTION
**Pull request by ErebusZ**
Addressing docker scout findings.

### CODE REVIEW

**Review by rabsondev — COMMENTED** [review]
Can you add the reason(s) for this change in the description?

**Review by BlueSquare1 — APPROVED** [review]
Please use a more meaningful title/description to indicate that versions are pinned to skip vulnerable versions.

**Comment by ErebusZ** [comment]
> Can you add the reason(s) for this change in the description? It's not clear why `setuptools` is `>=65.5.1`.

older version are flagged by docker scout as vulnerable.

**Review by amine3 — DISMISSED** [review]
Where is the list of findings check the finding? Add it to the PR description

**Comment by ErebusZ** [comment]
> Where is the list of findings check the finding? Add it to the PR description
its an extensive list, you can check it on docker hub.

File: Dockerfile [hunk 1]
```diff
@@ -3,6 +3,7 @@ FROM base as builder
 RUN mkdir /install
 WORKDIR /ostorlab
 COPY . /ostorlab
+RUN pip install -U pip==23.3
```
Comment (elyousfi5) [inline Dockerfile:6]: Why not ?
```suggestion
RUN pip install -U pip
```

Comment (3asm) [reply Dockerfile:6]: Bigger than.

File: setup.cfg [hunk 2]
```diff
@@ -50,7 +50,7 @@ install_requires =
     docker
     importlib-metadata; python_version<"3.8"
     jsonschema>=4.4.0
-    protobuf~=3.20
+    protobuf~=3.20.2
```
Comment (3asm) [inline setup.cfg:53]:
```suggestion
    protobuf~=3.20
```

File: setup.cfg [hunk 3]
```diff
@@ -78,12 +78,12 @@ exclude =
 
 # Add here agents requirements (semicolon/line-separated)
 agent =
-    Werkzeug==2.3.7
+    Werkzeug==2.3.8
```
Comment (3asm) [inline setup.cfg:81]:
```suggestion
    Werkzeug~=2.3
```

File: setup.cfg [hunk 4]
```diff
@@ -78,12 +78,12 @@ exclude =
...
     aio-pika
     flask
     importlib-metadata; python_version<"3.8"
     jsonschema>=4.4.0
-    protobuf==3.20.1
+    protobuf==3.20.2
```
Comment (3asm) [inline setup.cfg:86]:
```suggestion
    protobuf~=3.20
```

**Review by 3asm — CHANGES_REQUESTED** [review]

**Review by 3asm — APPROVED** [review]

**Review by mohsinenar — APPROVED** [review]

**Review by benyissa — COMMENTED** [review]
LGTM

File: setup.cfg [hunk 5]
```diff
@@ -99,7 +99,7 @@ google-cloud-logging =
 
 # Add here test requirements (semicolon/line-separated)
 testing =
-    setuptools
+    setuptools>=65.5.1
```
Comment (oussamaessaji) [inline setup.cfg:102]: why not latest?

**Review by 3asm — APPROVED** [review]

**Pull request merged**